src/neml2/models/solid_mechanics/ChabochePlasticHardening.cxx

```diff
@@ -33,7 +33,7 @@ register_NEML2_object(ChabochePlasticHardening);
 OptionSet
 ChabochePlasticHardening::expected_options()
 {
-  OptionSet options = FlowRule::expected_options();
+  OptionSet options = FredrickArmstrongPlasticHardening::expected_options();
   options.doc() +=
```

Comment:
Note the `operator+=`: Now that Chaboche derives from FredrickArmstrong, the class description should change.

Reply:
See https://reverendbedford.github.io/neml2/pr-preview/pr-166/syntax-models.html#chabocheplastichardening

Reply:
Ahh, now I get what you are doing.